util/devel/unstableWarningAnonymizer/unstableAnonScript.chpl

```diff
@@ -0,0 +1,52 @@
+import IO;
```

Comment:
Let's add a comment to the beginning of this file on how to use it

Reply:
Should I also add the copyright and license information that we have in many of our files ([like this](https://github.com/chapel-lang/chapel/blob/main/compiler/main/config.cpp)) since we plan on distributing it to users?
And is this a good directory for this script to be in or could there be a better place?

Reply:
Copyright is probably a good idea, yeah.  I'd probably put it in the `tools` directory, though